config/overlays/make-deploy/kustomization.yaml

```diff
@@ -3,3 +3,9 @@ kind: Kustomization
 namespace: opendatahub
 resources:
 - ../../base
+patchesStrategicMerge:
+- img_patch.yaml
+images:
+- name: controller
+  newName: quay.io/opendatahub/data-science-pipelines-operator
+  newTag: main
```

Comment:
Aren't you undoing what you did in https://github.com/opendatahub-io/data-science-pipelines-operator/pull/592 over here?

Reply:
see comment in PR body